hacks/hacks.mozilla.org.1.md

```diff
@@ -0,0 +1,118 @@
+# Dweb: Matrix
+
+## Author info: Full name, photo of yourself, and 1-2 sentence bio
+
+## What is Matrix?
+
+Matrix is an open standard for *interoperable*, *decentralised*, *real-time* communication over IP. It provides a standard HTTP API for publishing and subscribing to real-time data in specified channels, which means it can be used to power Instant Messaging, VoIP/WebRTC signalling, Internet of Things communication, and anything else that could be transmitted over HTTP. The most common use of Matrix today is as an Instant Messaging platform.
```

Comment:
"and anything else that can be expressed as JSON and needs to be transmitted in realtime over HTTP" perhaps?

(also, it'd be good if these were wordwrapped to aid review, but not the end of the world)

Reply:
oops. yeah I've been messing with VS Code settings, I guess this works for me but doesn't play nicely outside